Krivorotova Tatyana/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Cubes</title>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato:700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <div id="widget">
+            <div id="a" class="elem">
+                <img src="images/chicago.png">
+                <span>CHICAGO / 23&deg;</span>
+            </div>
+            <div id="b" class="elem">
+                <img src="images/seattle.png">
+                <span>SEATTLE / 34&deg;</span>
+            </div>
+            <div id="c" class="elem">
+                <img src="images/brooklyn_cloud.png">
+                <span>BROOKLYN / 55&deg;</span>
+            </div>
+            <div id="d" class="elem">
+                <img src="images/portland.png">
+                <span>PORTLAND / 27&deg;</span>
+            </div>
+            <div id="e" class="elem">
+                <img src="images/montue_cloud.png">
+                <div id="g">
+                    <div id="i">
+                        <div class="row">
```

Comment:
hmm, looks like elements from the next 3 rows below - logically is the one entity ( I mean that here is just 4 entities like `<span>MON</span><img/><span>22&deg;</span>` if I'm correct you need to change this markup to something like    
`<ul class=row>`
    `<li class="something with display: flex and flex-direction: column">`
        `<span>MON</span><img/><span>22&deg;</span>`
    `</li>` and so on